pkg/destroy/aws/aws.go

```diff
@@ -185,19 +185,23 @@ func (o *ClusterUninstaller) RunWithContext(ctx context.Context) ([]string, erro
 	// running on the cluster creating new resources while we are attempting to delete resources, which could leak
 	// the new resources.
 	ec2Client := ec2.New(awsSession)
+	timer := time.Now()
```

Comment:
Improve upon the name of this variable.
1) It is not a timer.
2) The name has no context about what it is used for.